script.js

```diff
@@ -0,0 +1,78 @@
+const sumFix = 0.1 + 0.2;
+console.log(`Result - ${sumFix.toFixed(1)}`);
+
+const login = prompt("Введіть логін");
+const password = prompt("Введіть пароль");
+
+const corectLogin = "admin";
```

Comment:
`corectLogin` and `corectPassword` can be initialized as constants(with capital letters)
Plus
```suggestion
const correctLogin = "admin";
```